src/platforms/hubspot/client.ts

```diff
@@ -93,15 +93,21 @@ const makeClient = () => {
     schema: z.ZodSchema,
     properties?: string[],
   ): requestFunctionType<FindObjectInput, TOutput> =>
-    request(({ id, associations }: FindObjectInput) => ({
-      url: `/crm/${API_VERSION}/${module}/${id}`,
-      method: 'GET',
-      query: shake({
-        properties: properties?.join(','),
-        associations: associations?.join(','),
+    request(
+      ({
+        id,
+        associations,
+        properties: requestedProperties,
+      }: FindObjectInput) => ({
+        url: `/crm/${API_VERSION}/${module}/${id}`,
+        method: 'GET',
+        query: shake({
+          properties: (requestedProperties ?? properties)?.join(','),
```

Comment:
what if there are too many for a query param?

Reply:
it will fail, though I think that should be handled by the caller.

One workaround I was thinking of doing was this --
- list all contacts (for example) w/o and properties and get the ids
- make a batch request on those ids with all the properties required

That way we'd be putting the properties in the JSON which can hold a lot more data.